src/main/java/dinkplugin/DinkPluginConfig.java

```diff
@@ -1004,4 +1023,70 @@ default String diaryNotifyMessage() {
         return "%USERNAME% has completed the %DIFFICULTY% %AREA% Achievement Diary, for a total of %TOTAL% diaries completed";
     }
 
+    @ConfigItem(
+        keyName = "gambleEnabled",
+        name = "Enable BA Gamble",
+        description = "Enable notifications for BA gambles",
+        position = 120,
+        section = gambleSection
+    )
+    default boolean notifyGamble() {
+        return false;
+    }
+
+    @ConfigItem(
+        keyName = "gambleSendImage",
+        name = "Send Image",
+        description = "Send image with the notification",
+        position = 121,
+        section = gambleSection
+    )
+    default boolean gambleSendImage() {
+        return true;
+    }
+
+    @ConfigItem(
+        keyName = "gambleInterval",
+        name = "Notify Interval",
+        description = "Interval between when a notification should be sent",
+        position = 122,
+        section = gambleSection
+    )
+    @Range(min = 1)
+    default int gambleInterval() {
+        return 100;
```

Comment:
I chose a higher default here compared to other intervals after bringing this up with the Casual BA admins. There was some concern that people would keep it at 1, even if we instructed them to set it to 100 or more (our rank-ups happen at 100 / 500 / 1000 gambles, so this interval seemed to be the most reasonable).